lib/api/route/streams.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+part 'streams.g.dart';
+
+/// https://zulip.com/api/get-stream-topics
+Future<GetTopicsResult> getStreamTopics(
```

Comment:
Name the result class to match the endpoint method, following the pattern of other endpoints:

```suggestion
Future<GetStreamTopicsResult> getStreamTopics(
```